modules/KoreBuild.Tasks/Utilities/DependencyVersionsFile.cs

```diff
@@ -75,16 +73,15 @@ public static DependencyVersionsFile Create(bool addOverrideImport)
             return file;
         }
 
-        public static bool TryLoad(string sourceFile, TaskLoggingHelper Log, out DependencyVersionsFile file)
+        public static bool TryLoad(string sourceFile, out DependencyVersionsFile file)
         {
             try
             {
                 file = Load(sourceFile);
                 return true;
             }
-            catch (InvalidProjectFileException ex)
+            catch
```

Comment:
Why remove the Log here? Seems worrying to catch everything, not log, then continue.

Reply:
The caller should take care of logging when the project fails to load.